### ISSUE TITLE
Broken `latest` tag

### ISSUE THREAD

**Issue by itavero** [ThePhaseless/Byparr#16]
**Describe the bug**
If I try to pull the docker image from ghcr.io, it can't find the correct manifest.
```bash
$ docker pull ghcr.io/thephaseless/byparr
Using default tag: latest
latest: Pulling from thephaseless/byparr
no matching manifest for linux/arm64/v8 in the manifest list entries
$ docker pull ghcr.io/thephaseless/byparr --platform linux/arm64
Using default tag: latest
latest: Pulling from thephaseless/byparr
no matching manifest for linux/arm64 in the manifest list entries
```

If I [browse the package on GitHub.com](https://github.com/ThePhaseless/Byparr/pkgs/container/byparr/309457513?tag=latest), I only see `unknown/unknown` and `linux/amd64` listed as architectures.

![image](https://github.com/user-attachments/assets/27248017-9ba5-4d21-b47f-b4262270da22)

**I've:**
- [x] - read Loop Warning on Readme.
- [x] - done the troubleshooting from Readme.
- [x] - checked if such issue already exists.

**Docker host (please complete the following information):**
 - OS: QTS (Linux 5.10.60-qnap)
 - Platform and model: QNAP TS-435XeU
 - Architecture: ARM64 (v8)

**Comment by ThePhaseless** [comment]
Oh, somehow docker does not auto-tag the image with platform on github arm64 runner.

**Comment by ThePhaseless** [comment]
I got GitHub to correctly tag the image, please check now

**Comment by fl4shback** [comment]
Hi @ThePhaseless, sorry for the ping, but it seems that this issue is back, the last images aren't tagged for arm releases.

**Comment by ThePhaseless** [comment]
Hm, I've changed the action yaml file for better cache handling, thx for the info, will inspect later today

**Comment by ThePhaseless** [comment]
let me know if v1.0.8 works for you after it is build

**Comment by fl4shback** [comment]
I was able to pull it with the latest-linux-arm64 tag but for some reason the healthcheck fails.
If I disable it in my docker-compose file it works though.

**Comment by Railline** [comment]
i can't pull now on linux/amd4 xD

![Image](https://github.com/user-attachments/assets/9e91bf28-4daa-4f26-bf90-a0efb8e197e6)

**Comment by ThePhaseless** [comment]
damn

**Comment by fl4shback** [comment]
It’s because the image is split in 2 « releases » and the latest tag is only pushed on the arm one.

**Comment by ThePhaseless** [comment]
yes, but it should merge in the last action step, which fails...

**Comment by ThePhaseless** [comment]
please use `main` branch for now, I will try to figure out this action mess

**Comment by ThePhaseless** [comment]
I've finally fixed it. `latest` tag now contains both amd64 and arm64. Both passed captcha on my main machine and arm vps
